### ISSUE TITLE
Crashing when attempting to load

### ISSUE THREAD

**Issue by wenzstev** [aditdamodaran/cloudy-day-club#3]
Was able to provide my Spotify credentials and authenticate, but since then the page won't load. Checking the console shows a few errors:
![cloudy day errors](https://user-images.githubusercontent.com/54607427/93827298-fbbb2180-fc36-11ea-86fb-0fbe101746e5.png)

**Comment by aditdamodaran** [comment]
Hey Steve (@wenzstev)!  Thank you for opening an issue! Would you mind letting me know what browser you're using (and whether you have cookies enabled on the site?). I'm trying to recreate this error.

**Comment by wenzstev** [comment]
Sure! I'm using Chrome, with cookies enabled. I deleted my cookie history and it didn't fix it. Also tried it on Edge and got the same error. Here it is in edge:

![edge-errors](https://user-images.githubusercontent.com/54607427/94050683-98e79880-fda4-11ea-917c-62f6d36a3941.png)

I have a Spotify Family account, maybe that could be what's wrong?

**Comment by aditdamodaran** [comment]
Hey Steve! (@wenzstev )

I think I fixed this bug! It looks like Spotify allows users to have playlists without an image: https://developer.spotify.com/documentation/web-api/reference/playlists/get-list-users-playlists/

The bug was caused by attempting to access an index of an array that was empty. Looks like two of your playlists might not have images. Kinda sad oversight on my part haha. I've patched it now by using a logo image for any playlists without an image. Lemme know if this works for you now!
